src/framework/ui/tab/tab.spec.tsx.snap

```diff
@@ -9,13 +9,12 @@ exports[`@tab: component checks * empty 1`] = `
   selected={false}
   style={
     Array [
-      undefined,
-      Object {},
       Object {
         "alignItems": "center",
         "flex": 1,
         "justifyContent": "center",
       },
+      undefined,
```

Comment:
Is this "custom" style?

Reply:
I guess this is a result of spreading `themedStyle`:
```
const {
  indicatorHeight,
  indicatorBorderRadius,
  indicatorBackgroundColor,
  ...containerParameters
} = source;
```

Where `containerParameters` can contain nothing